### ISSUE TITLE
postcss integration

### ISSUE THREAD

**Issue by bkinsey808** [alexpods/angular2-universal-starter#41]
If you are interested, I would love to see postcss (for processing css files) in this project. There are a lot of very interesting postcss plugins available nowadays, it would be great to have the power and flexibility of tapping into these. Some of the plugins that intrigue me are:

- for future syntax css: http://cssnext.io/
- for flexbox in older browsers: https://github.com/10up/flexibility (https://github.com/7rulnik/postcss-flexibility)

**Comment by alexpods** [comment]
@bkinsey808 Sorry for delay with answer, I was kinda busy these days. Of course postcss is must-have today. It's support was implemented in #44. I've added only one plugin by default: `postcss-cssnext`. I don't think many people need `flexibility` plugin. After all it's quite simple to add its support:

1. `npm install -D postcss-flexibility`
2. Add `require('postcss-flexibility')` [here](https://github.com/alexpods/angular2-universal-starter/blob/a6888e840c1d2ff385e84311fc539d09e6a220cc/webpack.config.js#L89)
